Master

barnabyrobson opened the pull request:
missing reverb.h in reverb.c

barnabyrobson committed:
vim ignore

ignore vim backups

barnabyrobson committed:
missing .h

missing reverb.h in reverb.c

barnabyrobson committed:
Revert cd55870..cae812e

This rolls back to commit cd5587045f0fec6109081af5ff852c0ff1773198.

dpiponi committed:
Added ladder.h, ladder.c

dpiponi commented:
## Added in ladder.[ch]

Dan

On Sat, Feb 21, 2015 at 8:32 PM, barnabyrobson notifications@github.com
wrote:

> ## missing reverb.h in reverb.c
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/dpiponi/Moodler/pull/1
> Commit Summary
> - vim ignore
> - missing .h
> 
> File Changes
> - _M_ .gitignore
>   https://github.com/dpiponi/Moodler/pull/1/files#diff-0 (1)
> - _M_ Moodler/reverb.c
>   https://github.com/dpiponi/Moodler/pull/1/files#diff-1 (1)
> 
> Patch Links:
> - https://github.com/dpiponi/Moodler/pull/1.patch
> - https://github.com/dpiponi/Moodler/pull/1.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/dpiponi/Moodler/pull/1.

barnabyrobson committed:
Revert "Revert cd55870..cae812e"

This reverts commit 24e71797aa5b2f9f59cf6890c1fcb638040c4bd4.

barnabyrobson committed:
Merge branch 'master' of https://github.com/dpiponi/Moodler

because you gotta merge

barnabyrobson committed:
Adding portaudio to makefile.

barnabyrobson committed:
Rough patch for portaudio backend.

barnabyrobson committed:
clean up

Commit:
Makefile support for Portaudio install in custom location

if Portaudio is installed in a location not in the standard include /
lib dirs you need to set PORTAUDIO_DIR in the environment.

Commit:
Merge remote-tracking branch 'dpiponi/master'

barnabyrobson committed:
Merge pull request #1 from barnabyrobson/portaudio

Merge Portaudio branch

Commit:
Merge remote-tracking branch 'origin/master'

Conflicts:
	Moodler/Makefile

Commit:
Merge branch 'master' into oscilloscope-01

Commit:
Make callback more efficient.

Removed some of the multiplies to get at the stereo samples and instead
just loop over all samples.

Commit:
Consistency around NUM_CHANNELS

Commit:
Add comments, remove cruft.

Just adding some stuff as I learn about it .. might be helpful one day.

Commit:
Audio callback sends audio via UDP.

Using UDP we send the summed audio buffer to a (currently) hard coded
port. This will allow clients to connect to the audio stream.

Commit:
Comment about summing voices (states).

Please read the comment.

Commit:
Adding Oscilloscope.

This is a first implementation of the oscilloscope. It’s actually just
a live wave form viewer right now and doesn’t have any real features of
an oscilloscope.

The implementation is based on UDP sending of audio data from a fixed
port in Moodler to a fixed port in the Oscilloscope. We could change
that to not require that the oscilloscope has a fixed port but that’s
not too important for the first working version.

For the graphics it uses GLFW and OpenGL; a very simple vertex and
fragment shader to draw the curves. I’m sure I’m doing something wrong
..

barnabyrobson committed:
Merge pull request #2 from barnabyrobson/oscilloscope-01

Oscilloscope 01

Commit:
Merge remote-tracking branch 'dpiponi/master'

Conflicts:
	Moodler/Makefile
